Description: Fix EH Trampoline
 Don't use CXXFLAGS when compiling eh_trampoline.cc
 The way that we were doing this didn't handle multiple flags and we
 actually don't want the user to override these flags because that file
 needs to be compiled in a very specific way.
 .
 libobjc2 (2.1-1) unstable; urgency=low
 .
   * Initial release
Author: David Chisnall <github@theravensnest.org>

---
Last-Update: 2023-09-09

--- libobjc2-2.1.orig/CMakeLists.txt
+++ libobjc2-2.1/CMakeLists.txt
@@ -317,7 +317,7 @@ if (ENABLE_OBJCXX)
 			endif()
 		endif ()
 		add_custom_command(OUTPUT eh_trampoline.s
-			COMMAND ${CMAKE_CXX_COMPILER} ${CMAKE_CXX_FLAGS} -fPIC -S "${CMAKE_SOURCE_DIR}/eh_trampoline.cc" -o - -fexceptions -fno-inline | sed "s/__gxx_personality_v0/test_eh_personality/g" > "${CMAKE_BINARY_DIR}/eh_trampoline.s"
+			COMMAND ${CMAKE_CXX_COMPILER} -fPIC -S "${CMAKE_SOURCE_DIR}/eh_trampoline.cc" -o - -fexceptions -fno-inline | sed "s/__gxx_personality_v0/test_eh_personality/g" > "${CMAKE_BINARY_DIR}/eh_trampoline.s"
 			MAIN_DEPENDENCY eh_trampoline.cc)
 		list(APPEND libobjc_ASM_SRCS eh_trampoline.s)
 		list(APPEND libobjc_CXX_SRCS objcxx_eh.cc)
